style.css

```diff
@@ -81,32 +81,68 @@ html, body {
 .header__count {
     padding: 3px 10px 0 5px;
 }
-.logo__text {
-    margin: 0 5px 10px 10px;
-    font-family: var(--logo-font);
-    font-weight: 200;
-    color: var(--main-color);
-    z-index: 5;
-}
 .header__body {
-    position: relative; 
     display: flex;
     justify-content: space-between;
-    height: 80px;
     align-items: center;
 }
 .header__container {
-    position: relative; 
     display: flex;
 }
-.header__burger {
-    display: none;
+.header__menu {
+    position: fixed;
+    top: -100%;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    padding: 70px 10px 20px;
+    background-color: var(--main-bg-color);  
+    transition: 0.5s;
+    overflow: auto;
 }
-.header__list {
-    display: flex;  
+.header__menu.active {
+    top: 0;
+}
+.header__burger {
     position: relative;
-    z-index: 2;
-    padding: 0 5px 0 0;
+    width: 30px;
+    height: 20px;
+    z-index: 3;
+    margin-top: 3px;
+}
+.header__burger:before, .header__burger:after {
+    content: '';
```

Comment:
Is this necessary for hamburger button?

Reply:
Yes, this is necessary to draw the upper and lower lines of burger